runtime/vm/NativeHelpers.cpp

```diff
@@ -149,6 +149,17 @@ convertCStringToByteArray(J9VMThread *currentThread, const char *cString)
 	return result;
 }
 
+UDATA *
+convertToNativeArgArray(J9VMThread *currentThread, j9object_t argArray, UDATA *javaArgs)
+{
+	UDATA argCount = (UDATA)J9INDEXABLEOBJECT_SIZE(currentThread, argArray);
+	UDATA *result = javaArgs;
+	if (NULL != result) {
```

Comment:
The doc says this can't be NULL.

Reply:
The code there has been updated to remove the NULL check.